app/src/main/java/com/example/trashminder/presentation/newReminder/NewReminderFragment.kt

```diff
@@ -234,24 +234,35 @@ class NewReminderFragment : Fragment() {
         val day = calendar.get(Calendar.DAY_OF_MONTH)
         val hour = calendar.get(Calendar.HOUR_OF_DAY)
         val minutes = calendar.get(Calendar.MINUTE)
```

Comment:
Could this be moved to the viewmodel? Keep all the date/time variables + formatting and calculations in there?